proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -2264,6 +2354,71 @@
         "border-radius": {
           "$value": "{basis.border-radius.md}",
           "$comment": "FIXME: This value should be selected from actual purmerend.nl designs. Could not find a good example."
+        },
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "color": {
+          "$type": "color",
+          "$value": "{basis.color.text.text-1}"
+        },
+        "error": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.error.bg-1}"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.error.text-1}"
+          }
+        },
+        "ok": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.success.bg-1}"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.success.text-1}"
+          }
+        },
+        "warning": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.bg-1}"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.text-1}"
+          }
+        },
+        "info": {
+          "background-color": {
+            "$value": "{basis.color.info.bg-1}"
+          },
+          "color": {
+            "$value": "{basis.color.info.text-1}"
+          }
+        }
+      }
+    }
+  },
+  "components/color-sample": {
+    "utrecht": {
+      "color-sample": {
+        "background-color": {
+          "$comment": "FIXME: what color from the theme can we use here?",
```

Comment:
Deze kleur moet simpelweg **niet ingesteld** zijn, de achtergrondkleur is gebaseerd op de input van de component.